test/integration/ssl_integration_test.cc

```diff
@@ -15,9 +15,10 @@
 #include "integration.h"
 #include "utility.h"
 
-namespace Envoy {
 using testing::Return;
 
+namespace Envoy {
+
```

Comment:
super small nit: del new line between namespace decls  when nothing in between (same in other files)